File: common/src/main/java/com/server/computerscience/question/common/service/ExternalQuestionService.java
package com.server.computerscience.question.common.service;

import com.server.computerscience.chatbot.dto.response.ChatGptFileUploadResponseDto;
import com.server.computerscience.question.common.dto.request.RequestQuestionCommandDto;
import org.springframework.stereotype.Service;


@Service
public interface ExternalQuestionService {
	ChatGptFileUploadResponseDto sendQuestionToExternal(RequestQuestionCommandDto requestQuestionCommandDto);
}

File: application/src/main/java/com/study/computerscience/question/major/admin/service/ExternalQuestionService.kt
package com.study.computerscience.question.major.admin.service

import com.study.computerscience.chatbot.dto.response.ChatGptFileUploadResponseDto
import com.study.computerscience.question.common.dto.request.RequestQuestionCommandDto

@org.springframework.stereotype.Service
interface ExternalQuestionService {
	fun sendQuestionToExternal(requestQuestionCommandDto: RequestQuestionCommandDto): ChatGptFileUploadResponseDto?
}
